scripts/predeploy.sh

```diff
@@ -3,7 +3,4 @@
 set -e -x
 
 yarn assets
-gzip -S .cgz $(find public/assets -name '*.css')
-gzip -S .jgz $(find public/assets -name '*.js')
-bucket-assets --bucket $S3_BUCKET
-heroku config:set ASSET_MANIFEST=$(cat manifest.json) --app=$APP_NAME
+yarn upload-to-s3
```

Comment:
There's no need to gzip locally -- all of this can be taken care of in Cloudfront settings:

1) find your distro, and click the Behaviors tab

<img width="342" alt="Screen Shot 2019-04-26 at 8 54 01 PM" src="https://user-images.githubusercontent.com/236943/56844684-2437c400-6869-11e9-9840-04e9aed54325.png">

2) Hit edit
3) Flip on the automatic compression setting: 

<img width="427" alt="Screen Shot 2019-04-26 at 8 53 49 PM" src="https://user-images.githubusercontent.com/236943/56844688-374a9400-6869-11e9-921d-dbb2db75ec32.png">

Reply:
👍